index.bs

```diff
@@ -2584,7 +2583,7 @@ Starting an {{XRSessionMode/"inline"}} session does not implicitly carry the sam
 To determine if an <dfn>inline session request is allowed</dfn> for a given |global object| the user agent MUST run the following steps:
 
   1. If the session request contained any [=required features=] or [=optional features=] and the request was not made while the |global object| has [=transient activation=] or when [=launching a web application=], return `false`.
-  1. If the requesting document is not [=responsible=], return `false`.
+  1. If |global| is not a {{Window}}, return `false`.
```

Comment:
```suggestion
  1. If the |global object| is not a {{Window}}, return `false`.
```

Reply:
This doesn't  feel like an equivalent statement to me, but it does appear to follow the linked guidance, so LGTM with the above correction.